project/Dependencies.scala

```diff
@@ -20,31 +19,31 @@ object Dependencies {
   }
 
   object Logback {
-    private val version = "1.2.11"
+    private val version = "1.4.11"
     val core            = "ch.qos.logback" % "logback-core"    % version
     val classic         = "ch.qos.logback" % "logback-classic" % version
   }
 
   object Slf4j {
-    private val version    = "1.7.36"
+    private val version    = "2.0.9"
     val api                = "org.slf4j" % "slf4j-api"        % version
     val `log4j-over-slf4j` = "org.slf4j" % "log4j-over-slf4j" % version
   }
 
   object Cats {
-    private val version = "2.8.0"
+    private val version = "2.10.0"
     val core            = "org.typelevel" %% "cats-core" % version
     val laws            = "org.typelevel" %% "cats-laws" % version
   }
 
   object CatsEffect {
-    private val version = "3.4.8"
+    private val version = "3.5.2"
```

Comment:
Let's not update it above the version used in the corresponding version of `cats-helper` (`3.4.8` at the moment). They've introduced some implicit but significant changes related to `async` method in `3.5.0` and I'd be cautious to introduce it like that.